depthai_helpers/arg_manager.py

```diff
@@ -52,12 +56,20 @@ def parse_args():
                         help="Disparity / depth median filter kernel size (N x N) . 0 = filtering disabled. Default: %(default)s")
     parser.add_argument('-lrc', '--stereo_lr_check', action="store_true",
                         help="Enable stereo 'Left-Right check' feature.")
+    parser.add_argument('-ext', '--extended_disparity', action="store_true",
+                        help="Enable stereo 'Extended Disparity' feature.")
+    parser.add_argument('-sub', '--subpixel', action="store_true",
+                        help="Enable stereo 'Subpixel' feature.")
+    parser.add_argument("-ff", "--full_fov_nn", default=False, action="store_true",
+                        help="Full RGB FOV for NN, not keeping the aspect ratio")
     parser.add_argument("-scale", "--scale", default=1.0, type=float,
                         help="Scale factor for the output window. Default: %(default)s")
+    parser.add_argument("-cm", "--color_map", default="JET", choices=color_maps, help="Change color map used to apply colors to depth/disparity frames. Default: %(default)s")
     parser.add_argument('-sbb', '--spatial_bounding_box', action="store_true",
                         help="Display spatial bounding box (ROI) when displaying spatial information. The Z coordinate get's calculated from the ROI (average)")
     parser.add_argument("-sbb-sf", "--sbb_scale_factor", default=0.3, type=float,
```

Comment:
sbb_bf for consistency with other parameter namings

Reply:
same for cnn-size

Reply:
Agreed, also updated `--no_debug` flag to be consistent